src/app/backend/resource/generic/controller.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package owner
+package generic
```

Comment:
In general, it is not a good practice to use names that a biased from a different context. In this case controller is a pattern and also a domain object. Not so nice, but maybe ok since it is an existing domain term and we have no better alternative.

I was wondering, if we want to add some more generic or reflective methods to this package. On the other hand we could rename once this is the case...

Unless you happen to have an alternaive idea, I would agree to go with controller